### ISSUE TITLE
Feature: add different ways of looking at data

### ISSUE THREAD

**Issue by MelSumner** [MelSumner/a11y-automation#433]
Right now, users browse by potential violation and see what WCAG Success Criteria (SC) are applicable and what automated tests, if any, are available. 

Another feature that would be nice is the ability to browse by SC and see all violations that pertain to that particular SC. 

Finally, it would be nice to be able to offer some search and filter functionality.
